doc(release): prepare release notes for OSS 23.10 (#3205)
centreon/centreon-documentation: versioned_docs/version-23.10/releases/centreon-os.mdx

## Centreon Web

### 23.10.11

Release date: `April 9, 2024`

<details open>
  <summary>Enhancements</summary>

- [API] Added dashboards list ordering possibilities for creator, creation date and update date.

</details>

<details>
  <summary>Bug fixes</summary>

> - [API] Fixed an issue with categories in POST service and service template.

Reviewer: ```suggestion
- [API] Fixed an issue where categories did not appear in service and service template creation responses.
```

> - [API V1] Fixed an issue on centreon_realtime_hosts, when get/filter by criticality.

Reviewer: ```suggestion
- [API V1] Fixed an issue that occurred when filtering Resources Status by criticality.
```

> - [Resource Status] Fixed a bug occuring when user cleared search bar.

Reviewer: ```suggestion
- [Resource Status] Fixed a bug occurring when users cleared the search bar.
```

> - [Share] Fixed error occurring while granting viewer rights as an administrator.

Reviewer: ```suggestion
- [Share] Fixed an error occurring while granting viewer rights as an administrator.
```

## Centreon Gorgone

### 23.10.6

Release date: `April 9, 2024`

<details>
  <summary>Bug fixes</summary>

> - Fixed a bug on service auto discovery where only half of the host was correctly discovered.

Reviewer: ```suggestion
- Fixed a bug affecting service auto discovery, where only half of the host was correctly discovered.
```
